sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/azure_appconfiguration_client.py

```diff
@@ -70,12 +87,13 @@ def _create_appconfig_pipeline(self):
         ]
 
         return Pipeline(
-            RequestsTransport(configuration=self.config), policies  # Send HTTP request using requests
+            RequestsTransport(configuration=self.config),
```

Comment:
We should use whatever transport is passed in to us, falling back to a new `RequestsTransport` instance...